### DISCUSSION TITLE
如何设置首页文章折叠而不是展开

### DISCUSSION THREAD

**Discussion by wayner6** [ppoffice/hexo-theme-icarus#1199] [Q&A]
如图所示，我的网站的文章是展开的状态，如何设置为您那样折叠的形式并且有个`read more`按钮呢？ 
我的网站
![image](https://user-images.githubusercontent.com/88063517/230752127-ab5fe21a-143f-42bb-8169-8a7618de188b.png)
您的网站
![image](https://user-images.githubusercontent.com/88063517/230752168-b125ea9f-b657-48b6-97ae-766f7b489d23.png)

**Comment by ppoffice** [comment]
https://github.com/ppoffice/hexo-theme-icarus/issues?q=is%3Aissue+read+more+

**Comment by wayner6** [comment]
谢谢，采用这个插件解决了 https://github.com/ashisherc/hexo-auto-excerpt
